### ISSUE TITLE
Accuracy not increasing for 1 class segmentation

### ISSUE THREAD

**Issue by silvernox-dev** [divamgupta/image-segmentation-keras#333]
I want to train segmentation only on 1 class, so I have set ignore_zero_class as True. And I can see in the code that it still picks crossentropy loss. Not sure what is going wrong here. Any help will be be appreciated.

**Comment by varungupta31** [comment]
Were you able to find a solution?
Facing similar issues.

**Comment by silvernox-dev** [comment]
Yes, after a few changes it started working for me. Please have a look at the solution here - https://github.com/divamgupta/image-segmentation-keras/issues/334.
